1559691744941	Sync.LogManager	DEBUG	Flushing file log
1559691744960	Sync.LogManager	DEBUG	Log cleanup threshold time: 1558827744960
1559691745193	Sync.LogManager	DEBUG	Done deleting files.
1559693633212	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559693633310	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1559693633310	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1559693633310	Sync.Service	DEBUG	User-Agent: Firefox/67.0 (Intel Mac OS X 10.14) FxSync/1.69.0.20190516215225.desktop
1559693633310	Sync.Service	INFO	Starting sync at 2019-06-05 02:13:53 in browser session DLDB2kz0MYXZ
1559693633310	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1559693633313	Sync.Status	INFO	Resetting Status.
1559693633313	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1559693633314	Sync.BrowserIDManager	INFO	Getting keys
1559693633314	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1559693633315	FirefoxAccounts	DEBUG	enter getAssertion()
1559693633315	FirefoxAccounts	DEBUG	getCertificateSigned: true true
1559693633326	Services.Common.RESTRequest	TRACE	local time, offset: 1559693633326, 2704
1559693633329	Services.Common.RESTRequest	DEBUG	POST request to https://api.accounts.firefox.com/v1/certificate/sign
1559693633329	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1559693633329	Services.Common.RESTRequest	TRACE	HTTP Header content-type: application/json
1559693633329	Services.Common.RESTRequest	TRACE	HTTP Header accept-language: de,en-us,en
1559693633329	Services.Common.RESTRequest	DEBUG	POST Length: 888
1559693633329	Services.Common.RESTRequest	TRACE	POST Body: {"publicKey":{"algorithm":"DS","y":"97CCB5D15873E4314736CEE2FA7A8D1099F91BB4958CE1B00C95A21684A0E99304028A4A799F22FB77054EF4D5FDC5609FDD5E896C90B7616D0D90DE4CCF11092DF98DD79753C3B6B84788BAE14F4FFD2061A66A93FDF79977157517FA6004D85DF599930BD1485F6035D148CA2D2A353BDFA6E44E48761B603604E66715852E","p":"FF600483DB6ABFC5B45EAB78594B3533D550D9F1BF2A992A7A8DAA6DC34F8045AD4E6E0C429D334EEEAAEFD7E23D4810BE00E4CC1492CBA325BA81FF2D5A5B305A8D17EB3BF4A06A349D392E00D329744A5179380344E82A18C47933438F891E22AEEF812D69C8F75E326CB70EA000C3F776DFDBD604638C2EF717FC26D02E17","q":"E21E04F911D1ED7991008ECAAB3BF775984309C3","g":"C52A4A0FF3B7E61FDF1867CE84138369A6154F4AFA92966E3C827E25CFA6CF508B90E5DE419E1337E07A2E9E2A3CD5DEA704D175F8EBF6AF397D69E110B96AFB17C7A03259329E4829B0D03BBC7896B15B4ADE53E130858CC34D96269AA89041F409136C7242A38895C9D5BCCAD4F389AF1D7A4BD1398BD072DFFA896233397A"},"duration":21600000}
1559693633751	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1559693633814	Services.Common.RESTRequest	TRACE	onStartRequest: POST https://api.accounts.firefox.com/v1/certificate/sign
1559693633814	Services.Common.RESTRequest	TRACE	Channel for POST https://api.accounts.firefox.com/v1/certificate/sign returned status code 2152398878
1559693633814	Services.Common.RESTRequest	DEBUG	POST https://api.accounts.firefox.com/v1/certificate/sign failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1559693633814	Services.Common.RESTRequest	TRACE	POST body: 
1559693633814	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:434
1559693633814	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 558"  data: no] Stack trace: get status()@resource://services-common/rest.js:558
request()@resource://services-common/hawkclient.js:224
1559693633815	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 573"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:573
request()@resource://services-common/hawkclient.js:227
1559693633815	Hawk	DEBUG	(Response) /certificate/sign: code: null - Status text: null
1559693633815	Services.Common.RESTResponse	TRACE	Processing response headers.
1559693633815	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 605"  data: no] Stack trace: get headers()@resource://services-common/rest.js:605
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:288
request()@resource://services-common/hawkclient.js:234
1559693633815	Services.Common.RESTResponse	TRACE	Processing response headers.
1559693633815	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 605"  data: no] Stack trace: get headers()@resource://services-common/rest.js:605
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:288
request()@resource://services-common/hawkclient.js:235
1559693633815	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 573"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:573
_constructError()@resource://services-common/hawkclient.js:116
request()@resource://services-common/hawkclient.js:240
1559693633815	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 558"  data: no] Stack trace: get status()@resource://services-common/rest.js:558
_constructError()@resource://services-common/hawkclient.js:117
request()@resource://services-common/hawkclient.js:240
1559693633816	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 558"  data: no] Stack trace: get status()@resource://services-common/rest.js:558
_constructError()@resource://services-common/hawkclient.js:118
request()@resource://services-common/hawkclient.js:240
1559693633816	Services.Common.RESTResponse	TRACE	Processing response headers.
1559693633816	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 605"  data: no] Stack trace: get headers()@resource://services-common/rest.js:605
_constructError()@resource://services-common/hawkclient.js:123
request()@resource://services-common/hawkclient.js:240
1559693633820	FirefoxAccounts	ERROR	error POSTing /certificate/sign: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1559693633821	FirefoxAccounts	ERROR	HAWK.signCertificate error: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1559693633821	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1559693633822	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1559693633822	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1559693633823	Sync.BrowserIDManager	ERROR	Failed to fetch a token for authentication: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 434\"  data: no]","message":null,"code":null,"errno":null}
1559693633828	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1559693633828	FirefoxAccounts	DEBUG	getOAuthToken enter
1559693633830	FirefoxAccounts	TRACE	getCachedToken returning cached token
1559693633830	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1559693633830	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1559693633830	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1559693633830	Services.Common.RESTRequest	TRACE	HTTP Header accept: application/json
1559693633830	Services.Common.RESTRequest	TRACE	HTTP Header if-none-match: "1d3cb6698a7af7b246a9837b970c1e724afce599-gzip"
1559693633834	Sync.Resource	WARN	GET request to https://sync-535-us-west-2.sync.services.mozilla.com/1.5/80856074/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1559693633834	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1559693633834	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1559693633834	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1559693633834	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1559693633838	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
